src/tests/cli_tests.py

```diff
@@ -796,12 +799,16 @@ def rnp_cleartext_signing_gpg_to_rnp(filesize):
     gpg_verify_cleartext(asc, KEY_SIGN_GPG)
     clear_workfiles()
 
-def gpg_supports_aead():
-    ret, out, err = run_proc(GPG, ["--version"])
-    if re.match(r'(?s)^.*AEAD:\s+EAX,\s+OCB.*', out):
-        return True
-    else:
-        return False
+def gpg_check_features():
+    global GPG_AEAD, GPG_2_3_1
+    _, out, _ = run_proc(GPG, ["--version"])
+    # AEAD
+    GPG_AEAD = re.match(r'(?s)^.*AEAD:\s+EAX,\s+OCB.*', out)
+    # Version 2.3 and up, dropping support of 64-bit algos
+    match = re.match(r'(?s)^.*gpg \(GnuPG\) ([0-9]+)\.([0-9]+)\.([0-9]+).*$', out)
+    if not match:
+        raise_err('Failed to parse GnuPG version.')
+    GPG_2_3_1 = (int(match.group(1)) > 2) or ((int(match.group(1)) == 2) and (int(match.group(2)) > 2) and (int(match.group(3)) > 0))
```

Comment:
This goes after 2.3.1 but the disabling of weak chiphers started in 2.3.0

Reply:
@andreasstieger thanks for noticing. Initially I tried to start with 2.3.0 but was hit with beta versions where option was not yet implemented. Now things should work for all GnuPG versions.